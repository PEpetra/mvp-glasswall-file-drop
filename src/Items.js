import React, { Component } from 'react'
import Item from './Item'

class Items extends Component {
  render(){
    return(
      <div class="items">
        {this.props.items.map(item => <Item item={item} />)}
      </div>
    )
  }
}

export default Items
