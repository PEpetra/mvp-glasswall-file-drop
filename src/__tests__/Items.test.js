import React from 'react';
import Enzyme, { shallow } from "enzyme";
import Items from '../Items';
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

const mockItems = [
    {children: ["Hello", "World"]},
    {children: ["How Are", "You"]}];

describe("Items", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<Items items={mockItems} />);
    expect(wrapper.exists()).toBe(true);
  });

  it("matches snapshot", () => {
    const wrapper = shallow(<Items items={mockItems} />);
    expect(wrapper).toMatchSnapshot();
  });
})
