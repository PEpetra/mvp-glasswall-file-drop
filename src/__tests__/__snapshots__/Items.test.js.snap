// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Items matches snapshot 1`] = `ShallowWrapper {}`;
